openmp/libomptarget/test/mapping/auto_zero_copy.cpp

```diff
@@ -0,0 +1,43 @@
+
+// RUN: %libomptarget-compilexx-generic
+// RUN: env HSA_XNACK=1 LIBOMPTARGET_DEBUG=1 %libomptarget-run-generic 2>&1 \
+// RUN: | %fcheck-generic -check-prefix=DEBUG -check-prefix=CHECK
+// REQUIRES: libomptarget-debug
+
+// UNSUPPORTED: aarch64-unknown-linux-gnu
```

Comment:
Instead of listing all architectures that are not supported, probably better to just require AMD

Reply:
Does the require look for the target arch or if we can compile for the target?

Reply:
Oh right. It just checks if we can compile. We will still need them here.